src/entities/swap/swaps/v2/auraBalSwaps/parseInputs.ts

```diff
@@ -0,0 +1,46 @@
+import { Address } from 'viem';
+import { Token } from '@/entities/token';
+import { AuraBalSwapKind, SwapQueryInput } from './types';
+import { supportedTokens, auraBalToken } from './constants';
+import { SwapKind } from '@/types';
+import { TokenAmount } from '@/entities/tokenAmount';
+
+export interface AuraBalSwapQueryInput {
+    kind: AuraBalSwapKind;
+    swapToken: Token;
+    inputAmount: TokenAmount;
+}
```

Comment:
Might want to move this declaration into `types.ts` next to `AuraBalSwapQueryOutput`